### PR TITLE
Kafka to Akka HTTP websockets example

### PR DESCRIPTION
**Pull request by jrudolph**
This is a working example but docs etc. are still missing. Not sure when I will get back to this but I leave it here for reference.

### CODE REVIEW

**Comment by ennru** [comment]
I have it almost integrated into the Alpakka Samples site...

**Comment by ennru** [comment]
Docs are included now. It would be nice to add a client app that can listen to the websocket.

**Comment by ennru** [comment]
Would be good to finish this.

**Comment by jrudolph** [comment]
> Would be good to finish this.

Indeed, any pointers to what is still missing?

**Comment by ennru** [comment]
I looked at it again and I believe this is good to be added. It is Java-only, but that's OK, I think.

File: docs/project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.31")
```
Comment (ennru) [inline docs/project/plugins.sbt:1]:
```suggestion
```

**Comment by jrudolph** [comment]
Thanks for completing the sample!

**Pull request merged**